pybossa/sched.py

```diff
@@ -218,15 +218,24 @@ def template_get_locked_task(project_id, user_id=None, user_ip=None,
             "Project {} - number of current users: {}"
             .format(project_id, user_count))
 
+        present_gold_task = not random.randint(0, 10)
```

Comment:
Do we want the decision for scheduling a golden task to be available outside of the `locked_scheduler`?  Perhaps at the level of `new_task`?  This would allow for logic that looks like:

```python
def new_task(...args):
    if want_golden_task():
        task = new_golden_task(...)
    return task or new_regular_task(...)
```

`want_golden_task` could be passed in to `new_task` as well...but I'm not sure how far it needs to go.